docs/advanced-topics/fips.asciidoc

```diff
@@ -0,0 +1,39 @@
+:page_id: fips
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Deploy a FIPS compatible version of ECK
+
+The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB 140-2), titled "Security Requirements for Cryptographic Modules" is a U.S. government computer security standard used to approve cryptographic modules. Since version 2.6 ECK offers a FIPS compliant image that is a drop-in replacement for the standard image.
+
+For the ECK operator, adherence to FIPS 140-2 is ensured by
+
+- Using FIPS approved / NIST recommended cryptographic algorithms
+- Compiling the operator using the link:https://github.com/golang/go/blob/dev.boringcrypto/README.boringcrypto.md[BoringCrypto] library for various cryptographic primitives.
+
+== Installation
+
+=== FIPS compliant installation using Helm
```

Comment:
might not want to use "FIPS compliant" wording - I'd prefer, FIPS-enabled
reason is, we can't really ensure "compliance" we can ensure that the operator uses the right libs and encryption in the (very) specific context of the operator, but customers requiring FIPS compliance on their ECK install (not just operator)  need to do much more than just use the FIPS-enabled operator.

so this is really just to avoid customer confusion.